src/commands/modping.ts

```diff
@@ -0,0 +1,160 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { getDB } from '@/db';
+import type { ChatInputCommandInteraction, GuildMember } from 'discord.js';
+import { ModPingSettings } from '@/models/modPingSettings';
+
+async function interactionHandler(interaction: ChatInputCommandInteraction): Promise<void> {
+    if (interaction.commandName !== 'mod-ping') {
+        return;
+    }
+
+    const roleId = getDB().get('roles.mod-ping');
+    if (!roleId) {
+        await interaction.reply({ content: `Missing mod-ping role ID!`, ephemeral: true });
+        return;
+    }
```

Comment:
This can also use `getRoleFromSettings()`. This would also mean you don't need to re-assign `role` later, which removes some error conditions that are currently handled with an else block.